### PR TITLE
feat(module:table): nzExpand supports custom icon

### PR DESCRIPTION
**Pull request by OriginRing**
fix: #7882

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: components/table/doc/index.zh-CN.md [hunk 1]
```diff
@@ -179,6 +179,7 @@ Table 组件同时具备了易用性和高度可定制性
 | --- | --- | --- | --- |
 | `[nzShowExpand]` | 是否显示展开按钮 | `boolean` | - |
 | `[nzExpand]` | 当前展开按钮状态，可双向绑定 | `boolean` | - |
+| `[nzExpandIcon]` | 自定义展开图标 | `TemplateRef<void>` | - |
```
Comment (simplejason) [inline components/table/doc/index.zh-CN.md:182]: Also needs `sting` type:
```html
nzExpandIcon="add"
```

File: components/table/doc/index.en-US.md [hunk 2]
```diff
@@ -177,11 +177,12 @@ Checkbox property
 
 Expand property
 
-| Property | Description | Type | Default |
-| -------- | ----------- | ---- | ------- |
-| `[nzShowExpand]` | Whether show expand icon | `boolean` | - |
+| Property | Description                           | Type | Default |
+| -------- |---------------------------------------| ---- | ------- |
```
Comment (simplejason) [inline components/table/doc/index.en-US.md:181]: No need more  dash `-` and spaces.

File: components/table/doc/index.zh-CN.md (hunk 1 above)
Comment (OriginRing) [reply components/table/doc/index.zh-CN.md:182]: 单string也可以通过ng-template实现，这里对齐了antd

**Comment by lw-like** [comment]
Is there another way to change expand icon? I'm strictly interested in this topic.

**Pull request merged**